spec/models/site_spec.rb

```diff
@@ -1,7 +1,55 @@
 RSpec.describe Site, type: :model do
+  let(:admin1) { FactoryGirl.create(:user, email: 'bob@was_here.net') }
+  let(:admin2) { FactoryGirl.create(:user, email: 'jane@was_here.net') }
+  let(:admin3) { FactoryGirl.create(:user, email: 'i@was_here.net') }
+
   describe ".instance" do
     it "is a singleton site" do
       expect(described_class.instance).to eq(described_class.instance)
     end
   end
+
+  describe ".admin_emails" do
+    subject { described_class.instance }
+    context "no admins exist" do
+      it "returns empty array" do
+        expect(subject.admin_emails).to eq([])
+      end
+    end
+    context "admins exist" do
+      before do
+        admin1.add_role :admin, subject
+        admin2.add_role :admin, subject
+      end
+      it "returns array of emails" do
+        expect(subject.admin_emails).to match_array([admin1.email, admin2.email])
+      end
```

Comment:
I see that `match_array` is not senstive to the order of elements in the array (I would argue that they should have called this matcher `match_set` instead), see
- http://www.rubydoc.info/github/rspec/rspec-expectations/RSpec/Matchers#contain_exactly-instance_method